src/mono/System.Private.CoreLib/src/System/MissingMemberException.Mono.cs

```diff
@@ -1,15 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-
-namespace System
-{
-    public partial class MissingMemberException
-    {
-#pragma warning disable IDE0060
-        internal static string FormatSignature(byte[] signature)
```

Comment:
Are there any tests to enable on Mono now?

Reply:
I didn't find if any coreclr test exercising this. The parsing of IL metadata signature should not depend on the runtime.